tests/util/test_yaml_extension.cpp

```diff
@@ -131,3 +164,57 @@ TEST(YamlEigenExtension, TsrTransforms)
   EXPECT_TRUE(right_cylinder.matrix().isApprox(expectedRightDefault.matrix()));
 }
+
+//==============================================================================
+TEST(YamlEigenExtension, AprilTagInJSON)
+{
+  std::string jsonString
+      = "{                                                             \n"
+        "  \"tag124\": {                                               \n"
+        "    \"resource\":                                             \n"
+        "      \"package://pr_ordata/data/objects/plastic_glass.urdf\",\n"
+        "    \"name\": \"plastic_glass\",                              \n"
+        "    \"offset\": [                                             \n"
+        "      [                                                       \n"
+        "        -0.0068393994632501,                                  \n"
+        "        -0.09054787493933,                                    \n"
+        "        -0.99586861832219,                                    \n"
+        "        0.14460904118031                                      \n"
+        "      ]                                                       \n"
+        "      ,                                                       \n"
+        "      [                                                       \n"
+        "        -0.9999766064057,                                     \n"
+        "        0.00052349074982687,                                  \n"
+        "        0.0068200145734176,                                   \n"
+        "        -0.00097599685803373                                  \n"
+        "        ]                                                     \n"
+        "      ,                                                       \n"
+        "      [                                                       \n"
+        "        -9.6209816943679e-5,                                  \n"
+        "        0.99589196617977,                                     \n"
+        "        -0.090549337061432,                                   \n"
+        "        -0.033644917605727                                    \n"
+        "      ]                                                       \n"
+        "      ,                                                       \n"
+        "      [                                                       \n"
+        "        0,                                                    \n"
+        "        0,                                                    \n"
+        "        0,                                                    \n"
+        "        1                                                     \n"
+        "      ]                                                       \n"
+        "    ]                                                         \n"
+        "  }                                                           \n"
+        "}                                                             \n";
```

Comment:
Nit: This line wrapping is terrible! :rage4: It is also odd that we're using a real configuration file here - that includes a crazy hard-coded transformation and references a file that will not exist on the build server (i.e. is inside the `pr_ordata` package) in an Aikido unit test.

We never try to load the file, so it's not technically wrong, but it does smell funny. 😬